target/linux/ramips/base-files/lib/upgrade/platform.sh

```diff
@@ -37,6 +37,7 @@ platform_do_upgrade() {
 
 	case "$board" in
 	hc5962|\
+	xiaomi,mir3p|\
```

Comment:
@iscilyas It would be nice to get this in alphabetical order finally

Reply:
Yeah. Good idea. Fixed.